packages/base/package.json

```diff
@@ -8,7 +8,7 @@
     "build:mock": "ENV=mock rollup --config rollup.config.js",
     "lint": "eslint --ext .js,.ts .",
     "clean": "rm -rf ./dist",
-    "prepublishOnly": "make ci",
+    "prepublishOnly": "npm run ci",
```

Comment:
There is no `ci` script
_[same for all packages]_

Reply:
sorry, my bad...adding it now